GitUp/Application/AppDelegate.m

```diff
@@ -411,6 +413,11 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
 #endif
 }
 
+- (void)_getUrl:(NSAppleEventDescriptor*)event withReplyEvent:(NSAppleEventDescriptor*)replyEvent {
+  NSString* url = [[event paramDescriptorForKeyword:keyDirectObject].stringValue stringByReplacingOccurrencesOfString:@"github-mac://openRepo/" withString:@""];
+  [self _cloneRepositoryFromURLString:url];
```

Comment:
The url for this repo is github-mac://openRepo/https://github.com/git-up/GitUp. Essentially I can just check to see that the prefix is 'github-mac://openRepo/' if that's good enough?